src/API/Google/Middleware.php

```diff
@@ -122,16 +123,15 @@ public function create_merchant_account(): int {
 	/**
 	 * Send a request to create a merchant account.
 	 *
-	 * @since 1.5.0
-	 *
-	 * @param string $name     Site name
+	 * @param string $name Site name
 	 * @param string $site_url Website URL
 	 *
 	 * @return int Created merchant account ID
 	 *
 	 * @throws Exception   When an Exception is caught or we receive an invalid response.
 	 * @throws InvalidTerm When the account name contains invalid terms.
 	 * @throws InvalidDomainName When the site URL ends with an invalid top-level domain.
+	 * @since 1.5.0
```

Comment:
Are these auto changed in your IDE? I don't think we've decided on a strict standard where to put the `@since` tag, but it would seem like they can be left alone if we aren't actually changing anything for those functions.

Reply:
Not sure if I get you on this one, this since was in older PRs

Reply:
That's what I meant, since the `@since` tags are from older PR's then why are thy being moved to a different location in this PR. Not a big deal I was just curious if that was intended or not.